base/docs/helpdb/Base.jl

```diff
@@ -923,13 +923,23 @@ Stacktrace:
 BoundsError
 
 """
-    invoke(f, types <: Tuple, args...)
+    invoke(f, argtypes::Type, args...)
```

Comment:
While we're at it we should probably include that `invoke` can work with `kwargs`:

```julia
julia> invoke(sort, Tuple{AbstractVector}, [3,1,2]; rev=true)
3-element Array{Int64,1}:
 3
 2
 1
```